products/sle/main.pm

```diff
@@ -931,6 +932,15 @@ sub load_x11tests {
         loadtest "x11/reboot_gnome";
     }
     loadtest "x11/desktop_mainmenu";
+    if (is_sles4sap() and !is_sles4sap_standard()) {
+        loadtest "sles4sap/sapconf";
+        loadtest "sles4sap/saptune";
+        loadtest "sles4sap/patterns";
+        if (get_var('SLES4SAP_MODE') !~ /wizard/) {
```

Comment:
```
if (get_var('SLES4SAP_MODE') !~ /wizard/  and get_var('NW')) {
      loadtest "sles4sap/nw_ascs_install"
}

```
would be more readable

Reply:
or even 
```
if (get_var('NW')) {
  loadtest "sles4sap/nw_ascs_install" if (get_var('SLES4SAP_MODE') !~ /wizard/ );
  loadtest "sles4sap/netweaver_ascs";
}
```
:)

Reply:
You're right. I had another 2 tests there (for TREX) like so:

```
if (get_var('SLES4SAP_MODE') !~ /wizard/) {
   loadtest "sles4sap/nw_ascs_install" if (get_var('NW'));
   loadtest "sles4sap/trex_install" if (get_var('TREX'));
}
loadtest "sles4sap/netweaver_ascs" if (get_var('NW'));
loadtest "sles4sap/trex" if (get_var('TREX'));
```

But then, deleted the TREX lines because the TREX tests won't be necessary, and leaving this `if` block with a single line.

Will update the code as suggested.